src/m365/teams/commands/team/team-clone.ts

```diff
@@ -107,12 +106,6 @@ class TeamsTeamCloneCommand extends GraphCommand {
     );
   }
 
-  #initOptionSets(): void {
-    this.optionSets.push(
-      { options: ['id', 'name'] }
```

Comment:
If this is no option set, I believe `id` and `name` should both be required options. We should update that in this file and in the documentation.

Also: the description of --name in the docs is a bit vague. Let's clarify that at the same time.

Reply:
You're right indeed. Let me fix that.